sklearn/manifold/t_sne.py

```diff
@@ -808,17 +812,27 @@ def _tsne(self, P, degrees_of_freedom, n_samples, random_state,
         # * early exaggeration with momentum 0.5
         # * early exaggeration with momentum 0.8
         # * final optimization with momentum 0.8
+
+        # TODO sklearn 0.20: remove this warning
```

Comment:
Not sure whether it should be 0.20 or 0.21 (we may want to apply the same policy than for deprecation messages, which last for two releases before change occurs).